scripts/upgrade.d/upgrade.11.sh

```diff
@@ -1,5 +1,8 @@
 gitlab_version="11.11.5"
 
+# There is no buster version for gitlab 11.X
+gitlab_x86_64_debian_verion="stretch"
```

Comment:
What about `version` instead of `verion` ?

Reply:
indeed :+1: